cmd/acra-translator/http_api/service.go

```diff
@@ -368,6 +384,9 @@ func (service *HTTPService) encryptOld(ctx *gin.Context) {
 // callOperationImplementation read body payload, pass it to implementation and render response
 func callOperationImplementation(ctx *gin.Context, f func(*gin.Context, []byte) (interface{}, HTTPError)) {
 	logger := logging.GetLoggerFromContext(ctx.Request.Context())
+	if ctx.Request.Method == http.MethodGet {
+		logger.Warningln("Has used deprecated method GET. Use method POST instead.")
```

Comment:
```suggestion
		logger.Warningln("Deprecated HTTP GET method was used. Please use HTTP POST method instead.")
```
If you commit this change (and maybe other cosmetic changes in a batch), consider adding `[skip ci]`  to commit title